application/repo-manager.jsx

```diff
@@ -347,10 +347,12 @@ export default class RepoNew extends React.Component {
             return  <div className="primary-app">
                         <h2>Import Repository</h2>
                         <p>
-                            Use the form below to import an existing repository under either your user
-                            or one of the organisations that you have write access to. Your existing
-                            files will no be overwritten, it is your responsibility to check that you
-                            are set up correctly. New files will, however, be added.
+                            Use this form to turn an unmanaged W3C organization repository into a W3C-managed repository. This tool does the following:
```

Comment:
```suggestion
                            Use this form to turn an unmanaged repository into a W3C-managed repository. This tool does the following:
```
(this makes it sound that it will bring the repo in to the W3C github organization otherwise)